pyreal/explainers/base.py

```diff
@@ -183,33 +163,27 @@ def transform_to_x_model(self, x_orig):
              DataFrame of shape (n_instances, x_model_feature_count)
                 x_orig converted to model-ready form
         """
-        if self.m_transformers is None and self.e_transformers is None:
-            return x_orig
-        if self.e_transformers is None:
-            return run_transformers(self.m_transformers, x_orig)
-        if self.m_transformers is None:
-            return run_transformers(self.e_transformers, x_orig)
-        return run_transformers(self.m_transformers, run_transformers(self.e_transformers, x_orig))
-
-    def transform_x_from_explain_to_model(self, x_explain):
+        am_transformers = _get_transformers(self.transformers, model=True)
+        return run_transformers(am_transformers, x_orig)
+
+    def transform_x_from_algorithm_to_model(self, x_algorithm):
         """
-        Transform x_explain to x_model, using the m_transformers
+        Transform x_algorithm to x_model, using the model transformers
 
         Args:
-            x_explain (DataFrame of shape (n_instances, x_orig_feature_count)):
+            x_algorithm (DataFrame of shape (n_instances, x_orig_feature_count)):
                 Input in explain space
 
         Returns:
              DataFrame of shape (n_instances, x_model_feature_count)
-                x_explain converted to model-ready form
+                x_algorithm converted to model-ready form
         """
-        if self.m_transformers is None:
-            return x_explain
-        return run_transformers(self.m_transformers, x_explain)
+        m_transformers = _get_transformers(self.transformers, algorithm=False, model=True)
```

Comment:
This should be `am_transformers`

Reply:
These are transformers where algorithm is False, and model is true. So I think `m_transformers` is most precise